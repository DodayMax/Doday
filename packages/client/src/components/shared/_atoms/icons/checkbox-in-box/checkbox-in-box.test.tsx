import * as React from 'react';
import { shallow } from 'enzyme';
import { Icons } from '@components';

describe('CheckboxInBox', () => {
  it('renders correctly without props', () => {
    const checkboxInBox = shallow(<Icons.CheckboxInBox />);

    expect(checkboxInBox).toMatchSnapshot();
  });
});