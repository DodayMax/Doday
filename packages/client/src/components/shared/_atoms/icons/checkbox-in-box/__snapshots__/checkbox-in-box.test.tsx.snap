// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckboxInBox renders correctly without props 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <defs>
          <path
            d="M4 2h16a2 2 0 0 1 2 2v16a2 2 0 0 1-2 2H4a2 2 0 0 1-2-2V4a2 2 0 0 1 2-2zm0 2v16h16V4H4z"
            id="checkboxInBox-1"
          />
        </defs>,
        <g
          fill="none"
          fillRule="evenodd"
        >
          <path
            d="M0 0h24v24H0z"
            fill="none"
          />
          <mask
            fill="#fff"
            id="checkboxInBox-2"
          >
            <use
              xlinkHref="#checkboxInBox-1"
            />
          </mask>
          <g
            fill="#000"
            mask="url(#checkboxInBox-2)"
          >
            <path
              d="M0 0h24v24H0z"
            />
          </g>
          <path
            d="M10.725 13.472L16.197 8 18 9.803l-7.275 7.274L6 12.352l1.803-1.803z"
            fill="#000"
            fillRule="nonzero"
          />
        </g>,
      ],
      "height": "20",
      "viewBox": "0 0 24 24",
      "width": "20",
      "xmlns": "http://www.w3.org/2000/svg",
      "xmlnsXlink": "http://www.w3.org/1999/xlink",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <path
            d="M4 2h16a2 2 0 0 1 2 2v16a2 2 0 0 1-2 2H4a2 2 0 0 1-2-2V4a2 2 0 0 1 2-2zm0 2v16h16V4H4z"
            id="checkboxInBox-1"
          />,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "d": "M4 2h16a2 2 0 0 1 2 2v16a2 2 0 0 1-2 2H4a2 2 0 0 1-2-2V4a2 2 0 0 1 2-2zm0 2v16h16V4H4z",
            "id": "checkboxInBox-1",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
        "type": "defs",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />,
            <mask
              fill="#fff"
              id="checkboxInBox-2"
            >
              <use
                xlinkHref="#checkboxInBox-1"
              />
            </mask>,
            <g
              fill="#000"
              mask="url(#checkboxInBox-2)"
            >
              <path
                d="M0 0h24v24H0z"
              />
            </g>,
            <path
              d="M10.725 13.472L16.197 8 18 9.803l-7.275 7.274L6 12.352l1.803-1.803z"
              fill="#000"
              fillRule="nonzero"
            />,
          ],
          "fill": "none",
          "fillRule": "evenodd",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M0 0h24v24H0z",
              "fill": "none",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <use
                xlinkHref="#checkboxInBox-1"
              />,
              "fill": "#fff",
              "id": "checkboxInBox-2",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "xlinkHref": "#checkboxInBox-1",
              },
              "ref": null,
              "rendered": null,
              "type": "use",
            },
            "type": "mask",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <path
                d="M0 0h24v24H0z"
              />,
              "fill": "#000",
              "mask": "url(#checkboxInBox-2)",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "d": "M0 0h24v24H0z",
              },
              "ref": null,
              "rendered": null,
              "type": "path",
            },
            "type": "g",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M10.725 13.472L16.197 8 18 9.803l-7.275 7.274L6 12.352l1.803-1.803z",
              "fill": "#000",
              "fillRule": "nonzero",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
        ],
        "type": "g",
      },
    ],
    "type": "svg",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <defs>
            <path
              d="M4 2h16a2 2 0 0 1 2 2v16a2 2 0 0 1-2 2H4a2 2 0 0 1-2-2V4a2 2 0 0 1 2-2zm0 2v16h16V4H4z"
              id="checkboxInBox-1"
            />
          </defs>,
          <g
            fill="none"
            fillRule="evenodd"
          >
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />
            <mask
              fill="#fff"
              id="checkboxInBox-2"
            >
              <use
                xlinkHref="#checkboxInBox-1"
              />
            </mask>
            <g
              fill="#000"
              mask="url(#checkboxInBox-2)"
            >
              <path
                d="M0 0h24v24H0z"
              />
            </g>
            <path
              d="M10.725 13.472L16.197 8 18 9.803l-7.275 7.274L6 12.352l1.803-1.803z"
              fill="#000"
              fillRule="nonzero"
            />
          </g>,
        ],
        "height": "20",
        "viewBox": "0 0 24 24",
        "width": "20",
        "xmlns": "http://www.w3.org/2000/svg",
        "xmlnsXlink": "http://www.w3.org/1999/xlink",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <path
              d="M4 2h16a2 2 0 0 1 2 2v16a2 2 0 0 1-2 2H4a2 2 0 0 1-2-2V4a2 2 0 0 1 2-2zm0 2v16h16V4H4z"
              id="checkboxInBox-1"
            />,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M4 2h16a2 2 0 0 1 2 2v16a2 2 0 0 1-2 2H4a2 2 0 0 1-2-2V4a2 2 0 0 1 2-2zm0 2v16h16V4H4z",
              "id": "checkboxInBox-1",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
          "type": "defs",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <path
                d="M0 0h24v24H0z"
                fill="none"
              />,
              <mask
                fill="#fff"
                id="checkboxInBox-2"
              >
                <use
                  xlinkHref="#checkboxInBox-1"
                />
              </mask>,
              <g
                fill="#000"
                mask="url(#checkboxInBox-2)"
              >
                <path
                  d="M0 0h24v24H0z"
                />
              </g>,
              <path
                d="M10.725 13.472L16.197 8 18 9.803l-7.275 7.274L6 12.352l1.803-1.803z"
                fill="#000"
                fillRule="nonzero"
              />,
            ],
            "fill": "none",
            "fillRule": "evenodd",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "d": "M0 0h24v24H0z",
                "fill": "none",
              },
              "ref": null,
              "rendered": null,
              "type": "path",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <use
                  xlinkHref="#checkboxInBox-1"
                />,
                "fill": "#fff",
                "id": "checkboxInBox-2",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "xlinkHref": "#checkboxInBox-1",
                },
                "ref": null,
                "rendered": null,
                "type": "use",
              },
              "type": "mask",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <path
                  d="M0 0h24v24H0z"
                />,
                "fill": "#000",
                "mask": "url(#checkboxInBox-2)",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "d": "M0 0h24v24H0z",
                },
                "ref": null,
                "rendered": null,
                "type": "path",
              },
              "type": "g",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "d": "M10.725 13.472L16.197 8 18 9.803l-7.275 7.274L6 12.352l1.803-1.803z",
                "fill": "#000",
                "fillRule": "nonzero",
              },
              "ref": null,
              "rendered": null,
              "type": "path",
            },
          ],
          "type": "g",
        },
      ],
      "type": "svg",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
